templates/vertical-standard/page.html.hbs

```diff
@@ -29,24 +29,28 @@
     partial for this page, you should not use it on any other page.  -->
     {{!-- {{> layouts/overlay-suggestions }} --}}
     <div class="Answers-container Answers-resultsWrapper">
-      {{!-- {{> templates/vertical-standard/markup/directanswer }} --}}
-      {{> templates/vertical-standard/markup/spellcheck }}
-      <div class="Answers-resultsHeader">
-        {{> templates/vertical-standard/markup/verticalresultscount }}
-        {{> templates/vertical-standard/markup/appliedfilters }}
-        {{!-- {{> templates/vertical-standard/collapsible-filters/markup/filterlink }} --}}
-        {{!-- {{> templates/vertical-standard/collapsible-filters/markup/viewresultsbutton }} --}}
+      <div class="Answers-resultsLeftContainer">
```

Comment:
maybe `Answers-leftSidebar` or something like that? I think calling it results container is a little confusing since it doesn't contain any results

Reply:
Now that I think about it, we should probably avoid left/right naming since we support rtl language as well. What do you think about `Answers-sidebar` and `Answers-maincontent`

Reply:
That makes sense! I like those names (probably camel case for mainContent?)